benchmarks/benchmarks.rb

```diff
@@ -3,18 +3,33 @@
 require "benchmark/ips"
 
 require "tempfile"
-require "memo_wise"
+
+github_memo_wise_path = Gem.loaded_specs["memo_wise"].full_gem_path
+
+GITHUB_MAIN = "GitHubMain"
+
+Dir.mktmpdir do |directory|
```

Comment:
MINOR: Can we have a comment explaining just that we are going to download the version of MemoWise from Github main branch into a temp directory here, to compare benchmarks against?